Source/JavaScriptCore/tools/Integrity.h

```diff
@@ -98,6 +98,10 @@ class Random {
 ALWAYS_INLINE static bool isSanePointer(const void* pointer)
 {
     uintptr_t pointerAsInt = bitwise_cast<uintptr_t>(pointer);
+#if CPU(ARM64)
```

Comment:
Don't you also want `&& CPU(ADDRESS64)`?

Reply:
Resolved in latest push.